src/llama_stack/providers/inline/agents/meta_reference/responses/utils.py

```diff
@@ -378,3 +379,65 @@ def extract_guardrail_ids(guardrails: list | None) -> list[str]:
             raise ValueError(f"Unknown guardrail format: {guardrail}, expected str or ResponseGuardrailSpec")
 
     return guardrail_ids
+
+
+def convert_file_search_tool_choice(chat_tool_names: list[str]) -> dict[str, Any]:
+    """Convert a responses tool choice of type file_search to a chat completions compatible function tool choice."""
+    tool_name = "knowledge_search"
+    if tool_name not in chat_tool_names:
+        raise ValueError("Knowledge search tool not found in chat tools")
+    return {"type": "function", "function": {"name": "knowledge_search"}}
+
+
+def convert_web_search_tool_choice(chat_tool_names: list[str]) -> dict[str, Any]:
```

Comment:
other than the mcp tool choice function, these other things are useless wrappers in my opinion and make the calling code harder to follow actually, what with the try: except: etc. all of that is unnecessary.